bless-good.sh

```diff
@@ -25,6 +25,9 @@ rm -f pinned-manifest.xml
 URL=`cat $BUILD_URL_FILE`
 wget $URL/$MANIFEST 2> /dev/null
 
+# Fix remote in manifest
+sed -i "s|\"..\"|\"https://android.googlesource.com/\"|g" $MANIFEST
```

Comment:
I'm a bit hesitant to be modifying the manifest files. The idea is this is what was used to build the images I'm testing with, so tweaking them feels like compromising them somehow.